fuzz/FuzzUtils.h

```diff
@@ -0,0 +1,17 @@
+#ifndef SIMDJSON_FUZZUTILS_H
+#define SIMDJSON_FUZZUTILS_H
+
+#include <cstdint>
+
+// view data as a byte pointer
+template <typename T> inline const std::uint8_t* as_bytes(const T* data) {
+  return static_cast<const std::uint8_t*>(static_cast<const void*>(data));
```

Comment:
Will `reinterpret_cast<const uint8_t*>` do the same thing? (Note: when I'm asking questions, it's usually because I don't know the answer (at least not for sure). I try not to *imply* a change, so if I don't say "we should [probably] change X." I'd rather not waste your time trying to guess what people mean when you could be coding!

Reply:
Sorry, missed to answer this! Yes, reinterpret_cast casts anything to anything (like a c-style cast). Therefore one tries to avoid it since it is so powerful. Also, i think the function name conveys meaning and is easier to read.